### ISSUE TITLE
Unable to Auto Attack after drinking Health Potion [$20 awarded]

### ISSUE THREAD

**Issue by 55Honey** [azerothcore/azerothcore-wotlk#5263]
Under some circumstances Health Potions cause a player to be unable to Auto Attack.

#### WHAT CLIENT DO YOU PLAY ON?

- [ ] enGB
- [x] enUS
- [ ] other (specify)

##### FACTION

- [x] Alliance (not tested)
- [x] Horde

##### CONTENT PHASE:

- [x] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39

##### CURRENT BEHAVIOUR:

If a player exits WoW while under the effects of a Troll's Blood Potion (multiple tested, all affected), using a Health Potion (multiple tested, all affected) after logging back in causes Auto Attack to not work.

##### EXPECTED BLIZZLIKE BEHAVIOUR:

Using a Health Potion, under any circumstances should not permanently disable a player's melee abilities.

##### STEPS TO REPRODUCE THE PROBLEM:

Setup: Add Troll's Blood Elixirs and Health Potions to inventory
```
.additem 3382 5
.additem 3388 5
.additem 3826 5
.additem 858 10
.additem 3928 10
.additem 22829 10
.additem 33447 10
```

Use one of the Troll's Blood Elixirs:
`/use Weak Troll's Blood Elixir`
`/use Strong Troll's Blood Elixir`
`/use Major Troll's Blood Elixir`

Exit game
`/exit`

Delete Cache folder: *optional*

Log into game

Now attempt to use one of the Health Potions (any will do)
`/use Lesser Healing Potion`
`/use Superior Healing Potion`
`/use Super Healing Potion`
`/use Runic Healing Potion`

If you use a potion out of combat you will notice the icon dulls (edit: ElvUI dulls icon, default UI has glowing border) as though it's been clicked but you will get no HP restored, and the cooldown will not start. Engaging a NPC in combat at this point, you will see that Auto Attack does not work.

Now `/exit` game, relaunch and attack another NPC, you will see it works fine.
While still in combat, use another Health Potion. Again, any will do.
You will again not get the effects of the potion, nor start the CD, but also, Auto Attack continue to work, but your weapons disappear.

##### EXTRA NOTES:

I found that if I engaged a NPC with the Warrior's Charge ability, Auto Attack functioned as normal, while this bug continued to affect Auto Attack on NPCs that were not engaged with Charge.

Instead of Troll's Blood Elixirs, I tried a few other items that also grant a HP5 buff.
 - Poached Sunscale Salmon
 - Book of Survival
 - Rune of Duty

Of the 3 items I tested, only Poached Sunscale Salmon actually gives a removable aura but none of them caused any issues with drinking a Health Potion.

Merely Logging out and Logging in does not clear the bugged state of a player, `/exit` is required to invoke and to clear this bug.

related: https://github.com/chromiecraft/chromiecraft/issues/386

##### AC HASH/COMMIT:

https://github.com/chromiecraft/azerothcore-wotlk/commit/ee1918a3e9bc355fbd6f8d58d77760fb61fdbff7

##### OPERATING SYSTEM:

Ubuntu 20.04

##### MODULES:

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-LevelUpReward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-CarbonCopy](https://github.com/55Honey/Acore_CarbonCopy)

##### OTHER CUSTOMIZATIONS:

None.

##### SERVER:

ChromieCraft

**Comment by 55Honey** [comment]
https://github.com/chromiecraft/chromiecraft/issues/386#issuecomment-841464846

I can confirm that it was resolved by canceling the [trollblood potion buff](https://wowgaming.altervista.org/aowow/?spell=3222) and relogging after that.
Relooging only with the buff still active did not help.

**Comment by 55Honey** [comment]
@FrancescoBorzi It seems like there  is a fix proposed already.

**Comment by ChromieCraftServer** [comment]
the proposed fix does not work, the 20$ bounty is free to be claimed by the first developer providing a working solution

**Comment by Azcobu** [comment]
Reopening this, as another report of this issue has been make on the CC tracker - see https://github.com/chromiecraft/chromiecraft/issues/1347

To verify, I ran through the steps given in the original post and reproduced the issue, so it appears either the original fix wasn't successful or a regression of some kind has occurred.

Here's a demo of the current error. Note that autoattack (the leftmost button on the bottom bar) is being pressed repeatedly, with no effect.

https://user-images.githubusercontent.com/81782124/128600669-dca054c1-59dd-4a86-a6cd-a46950337124.mp4

I also checked druid forms, and autoattacks were disabled for cat/bear as well. 

This is on 598e6768374a.
